ui/src/types/index.tsx

```diff
@@ -11,6 +11,8 @@ interface Model {
 export interface Story extends Model {
   title: string;
   content: string;
+  latitude: number;
+  longitude: number;
```

Comment:
Can we update this type to have all the story fields from the golang model https://github.com/uwblueprint/shoe-project/blob/main/internal/database/models/models.go#L20?